docker/ibm-cp4na-log-message.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+# This custom module is for printing some specified logs
+
+from ansible.module_utils.basic import AnsibleModule
+import requests
+
+def run_module():
```

Comment:
I think we probably need to unit test this Ansible module (use it in a playbook in a unit test) to verify it works and that we're getting the right result?

Reply:
> I think we probably need to unit test this Ansible module (use it in a playbook in a unit test) to verify it works and that we're getting the right result?

Sure Dan @dvaccarosenna . We will add unit test for this module as you suggested. We have tested the new module with command line ansible-playbook and also tested that module from Nimrod UI with Assembly. We are getting the desired results for the module when we run the attached playbook script which is having our custom module. But adding the unit test for this module is definitely a good idea.

[test_script.txt](https://github.com/IBM/ansible-lifecycle-driver/files/9021921/test_script.txt)